renderer/lib/location-history.js

```diff
@@ -12,14 +12,16 @@ LocationHistory.prototype.url = function () {
 }
 
 LocationHistory.prototype.current = function () {
-  return this._history[this._history.length - 1]
+  return this._current
 }
 
 LocationHistory.prototype.go = function (page, cb) {
   if (!cb) cb = noop
   if (this._pending) return cb(null)
 
-  console.log('go', page)
+  if (process.env.NODE_ENV !== 'test') {
```

Comment:
I think we should keep the tests. On second thought since this is the only place we do this, I wouldn't mind keeping it in, we can always change it later.